### PR TITLE
feat: samples for upcoming ELK tutorial

### PR DESCRIPTION
**Pull request by TheRealSpaceShip**
Tutorial contains:

- DaemonSet hack to increase virtual memory on nodes (for Elasticsearch)
 - CR to create Elasticsearch with ECK operator 
 - CR to create Kibana with ECK operator
 - CR to create Agents in fleet mode with ECK operator
 - Ingress object to setup LB with Google-managed SSL certificate

### CODE REVIEW

File: elastic-stack/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Elastic Stack on GKE
```
Comment (bourgeoisor) [inline elastic-stack/README.md:1]: Would you mind moving this tutorial to either `monitoring/elastic-stack-tutorial/` or `operations/elastic-stack-tutorial/`? I'm trying to get away from single samples as top-level directories, just for clean-up / sustainability purposes

**Review by bourgeoisor — CHANGES_REQUESTED** [review]
Just one quick comment--

**Review by bourgeoisor — APPROVED** [review]
LGTM.

Let me know when I can merge.

Please also create a follow-up PR once the tutorial is published.

**Comment by bourgeoisor** [comment]
@TheRealSpaceShip this is still ready to be merged whenever it's ready to go, just let me know

**Comment by TheRealSpaceShip** [comment]
@bourgeoisor this PR is done and ready to be merged.

**Review by bourgeoisor — CHANGES_REQUESTED** [review]
Oh, just noticed last-minute
Would you mind moving this from `monitoring/` to `observability/`?
Sorry for any confusion-- I'm in the middle of restructuring samples around :)

**Comment by TheRealSpaceShip** [comment]
> Oh, just noticed last-minute Would you mind moving this from `monitoring/` to `observability/`? Sorry for any confusion-- I'm in the middle of restructuring samples around :)

No problem. Done.

**Review by bourgeoisor — APPROVED** [review]
LGTM

**Comment by bourgeoisor** [comment]
@TheRealSpaceShip let me know when this is ready to be merged.

**Comment by TheRealSpaceShip** [comment]
@bourgeoisor ready to be merged.

**Pull request merged**